client/hooks/useMatchMedia.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+
+const useMatchMedia = (breakpoint: number) => {
+  const query = matchMedia(`(max-width: ${breakpoint}px)`);
```

Comment:
@colinvandergraaf Are you referring to the `query` variable? If it were moved inside the useEffect, then it could not be referenced in the dependency array. The current definition is intentional.